pkg/question/question.go

```diff
@@ -338,18 +344,24 @@ func AskIfEnterInstanceType(h *ec2helper.EC2Helper, defaultInstanceType string)
 		}
 	}
 
-	indexedOptions := []string{cli.ResponseYes, cli.ResponseNo}
-
-	optionsText := "1. I will enter the instance type\n2. I need advice given vCPUs and memory\n"
+	data := [][]string{{"Enter the instance type"}, {"Provide vCPUs and memory information for advice"},
+		{fmt.Sprintf("Use the saved default instance, [%s]", *defaultOption)}}
```

Comment:
The default instance type is not necessarily one that the user saved; I'd prefer to phrase it like this:

```suggestion
		{fmt.Sprintf("Use the default instance type, [%s]", *defaultOption)}}
```